plane/src/controller/proxy.rs

```diff
@@ -176,7 +176,13 @@ pub async fn handle_message_from_proxy(
         }
         MessageFromProxy::KeepAlive(backend_id) => {
             if let Err(err) = controller.db.backend().update_keepalive(&backend_id).await {
-                tracing::error!(?err, ?backend_id, "Error updating keepalive");
+                tracing::error!(?err, ?backend_id, ?node_id, "Error updating keepalive");
+
+                socket
```

Comment:
Maybe we should only send this if the error is `sqlx::Error::RowNotFound`?

Reply:
Good call. Updated.